File: Python3/Pizza_Box/pizza.py
import sys

class pizza:
    number = 0
    def __init__(self,name,price):
        self.price = price
        self.name = name
    def order(self,nums):
        self.number+=nums
    def cost(self):
        return self.price*self.number
    def get_num(self):
        if self.number>0:
            return True
        else:
            return False
    def get_name(self):
        return self.name
    def get_price(self):
        return self.price
    
if __name__=='__main__':
    tax = 0.3
    total_cost = 0
    
    pizzas = { }
    pizzas['A'] = pizza('pizza A',34.55)
    pizzas['B'] = pizza('pizza B',18.28)
    pizzas['C'] = pizza('pizza C',55.46)
    
    while True:
        print("\nBasket --------\n")
        if pizzas['A'].get_num()>0:
            print(f"{pizzas['A'].get_name()}  {pizzas['A'].cost()}")
        if pizzas['B'].get_num()>0:
            print(f"{pizzas['B'].get_name()}  {pizzas['B'].cost()}")
        if pizzas['C'].get_num()>0:
            print(f"{pizzas['C'].get_name()}  {pizzas['C'].cost()}")
        print(f"Total Cost {total_cost}\n---------------\n")

        print("Please select a pizza to add to your basket:")
        print(f"1: {pizzas['A'].get_name()} price: {pizzas['A'].get_price()}")
        print(f"2: {pizzas['B'].get_name()} price: {pizzas['B'].get_price()}")
        print(f"3: {pizzas['C'].get_name()} price: {pizzas['C'].get_price()}")

        choice = input("Enter your choice here >> ")
        choice = int(choice)

        if choice is 1:
            total_cost += pizzas['A'].get_price()*(1+tax)
            pizzas['A'].order(1)
        elif choice is 2:
            total_cost += pizzas['B'].get_price()*(1+tax)
            pizzas['B'].order(1)
        elif choice is 3:
            total_cost += pizzas['C'].get_price()*(1+tax)
            pizzas['C'].order(1)
        elif choice is 0:
            print("Terminating ...")
            sys.exit()
        else:
            print("You inserted incorrect number")
    
    

File: C/Pizza_Box/pizza.c
#include<stdio.h>

int main(){
    int choice = 0;
    double cost = 0,tax = 0.3;
    double pizza_price[3];
    pizza_price[0] = 50.99;
    pizza_price[1] = 23.99;
    pizza_price[2] = 18.33;
    char *pizza[3];
    pizza[0] = "A Pizza";
    pizza[1] = "B Pizza";
    pizza[2] = "C Pizza";
    int pizza_number[3];
        pizza_number[0] = 0;
        pizza_number[1] = 0;
        pizza_number[2] = 0;
    while (1){
        

        printf("\nBasket --------\n\n");
        if(pizza_number[0]>0) printf("%s  %.2f\n",pizza[0],pizza_price[0]*pizza_number[0]);
        if(pizza_number[1]>0) printf("%s  %.2f\n",pizza[1],pizza_price[1]*pizza_number[1]);
        if(pizza_number[2]>0) printf("%s  %.2f\n",pizza[2],pizza_price[2]*pizza_number[2]);
        printf("Total Cost %.2f\n---------------\n\n",cost);
        printf("Please select a pizza to add to your basket:\n");
        printf("1: %s price: %.2f\n",pizza[0],pizza_price[0]);
        printf("1: %s price: %.2f\n",pizza[1],pizza_price[1]);
        printf("1: %s price: %.2f\n",pizza[2],pizza_price[2]);
        printf("Enter your choice here >>\n");
        scanf("%d",&choice);

        
        switch (choice){
            case 1:
            cost += pizza_price[0]*(1+tax);
            pizza_number[0]+=1;
            break;
            case 2:
            cost += pizza_price[1]*(1+tax);
            pizza_number[1]+=1;
            break;
            case 3:
            cost += pizza_price[2]*(1+tax);
            pizza_number[2]+=2;
            break;
            case 0:
            printf("Exit\n");
            return 0;
            default:
            printf("You inserted incorrect number");
            break;
        }
    }

}